Apps/Playground/UWP/App.cpp

```diff
@@ -206,12 +209,16 @@ void App::OnWindowClosed(CoreWindow^ sender, CoreWindowEventArgs^ args)
     Uninitialize();
 }
 
-void App::OnPointerMoved(CoreWindow^, PointerEventArgs^ args)
+void App::OnPointerMoved(CoreWindow^ sender, PointerEventArgs^ args)
 {
     if (m_nativeInput != nullptr)
     {
         const auto& point = args->CurrentPoint->RawPosition;
+
         m_nativeInput->MouseMove(static_cast<int>(point.X), static_cast<int>(point.Y));
+        // Once move is handled, check for button changes
+        OnPointerPressed(sender, args);
+        OnPointerReleased(sender, args);
```

Comment:
Why do you need this? Maybe an explanation in the comment would be good.

Reply:
So this was part of a complex solution that I had to implement because UWP input doesn't work the same as Win32 input.  First of all, PointerPressed was only firing for the first button pressed so no other buttons were getting picked up.  PointerReleased had the same issue but with only firing when the last button is released.  For the button state tracking, that was implemented as the only way I've found to track when a button is pressed/released so that I can properly use the correct event without firing unnecessary up/down events.

Reply:
I can flesh out the comment explaining this and I'll also remove the sender since we don't use it.

Reply:
I don't think it's obvious why UWP doesn't send the events. A link to the documentation which says this would be great.

Reply:
Do we need to check `IsInContact`? The sample seems to check it, but I'm not sure we need it.

https://github.com/microsoft/Windows-universal-samples/blob/main/Samples/BasicInput/cpp/2-PointerPointProperties.xaml.cpp#L55-L60